java/old-change-notes/2021-02-25-local-temp-directory-information-disclosure.md

```diff
@@ -0,0 +1,4 @@
+lgtm,codescanning
```

Comment:
TODO: Convert to the new change note location / style (e.g. https://github.com/github/codeql/blob/main/java/ql/src/change-notes/2021-06-28-unsafe-cert-trust-query.md)

Reply:
Done!